### DISCUSSION TITLE
[Feature] Add search category/filter "Not Backed Up"

### DISCUSSION THREAD

**Discussion by irrigate-hanks** [immich-app/immich#3803] [Feature Request]
### The feature

I don't see an easy way to find the occasional one-off video or photo that refuses to back up for whatever reason. Something like this could help someone verify that they aren't unknowingly leaving something valuable unprotected.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2155
